rules.md

```diff
@@ -511,140 +422,177 @@ Ultimately, decisions on whether some output meets the requirements specified in
 
-### 4.4.1 Commission Bonuses
+> To set a Null Claim on their Commission Reward, an Admin should enter their Dash address as 'NULL' in the Claim in the [Incubator Accounting Spreadsheet](https://docs.google.com/spreadsheets/d/1mhXlo4ylqWLLSYN4MGiLWlp7Gq3jrsDt0kB701dwMNU/edit#gid=0)
 
-Each individual Admin's commission is variable based on the number of [Task Claims](#26-claims) they Approved in the previous 30 days (including "today"). The commissions that Admins can earn are tiered as follows:
+### 4.4.1 Commissions
 
 <table>
   <tr>
    <td><strong># of Claims Approved</strong>
    </td>
    <td><strong>Commission Rate</strong>
    </td>
+   <td><strong>Time of Award</strong>
+   </td>
   </tr>
   <tr>
-    <td>0 - 14
+    <td>Lead Strategist
+    </td>
+    <td>10% of delegated budget requests
     </td>
-    <td>10%
+    <td>When delegated budgets are paid out
     </td>
   </tr>
   <tr>
-    <td>15 - 29
+    <td>Strategist
+    </td>
+    <td>20% of task claim (10% if using Assistant Admin)
     </td>
-    <td>12%
+    <td>When approved tasks are paid out
     </td>
   </tr>
   <tr>
-    <td>30 +
+    <td>Assistant Admin
     </td>
-    <td>15%
+    <td>10% of task claim (half of Strategist's 20%)
     </td>
+    <td>When approved tasks are paid out
+    </td>    
   </tr>
-</table>  
+</table>
 
-Each Commission Bonus is based on the Tier that the Admin’s previous 30 days contributions fall into. The "next" tier of commission is effective immediately once an Admin crosses the threshold, and is effective for all claims as long as the Admin stays within the Tier.
+## 4.5 Governance
 
-For example:
+[Strategists](#46-strategists) are responsible for governance and the strategic direction of the Incubator.  They collectively form a group referred to as the strategic committee.
 
-- Admin Alice has processed 10 claims in the past 30 days (including today). If Alice processes 5 additional claims today, Alice will be eligible for a 12% commission on the 5th claim (and any additional claims).
-- Admin Bob has processed 34 claims in the past 30 days, but 7 of those were processed exactly 30 days ago. If Bob processes claims today, each of those will earn 15%. If Bob does not process at least 2 claims today, his commission rate will be 12% for claims processed tomorrow until his 30 day count is 30 or more again, at which point Bob will return to a 15% commission.
-- Admin Carol has processed 9 claims in the past 30 days. Today Carol processes 21 claims. For the first 5 claims Carol processes today the commission is 10%. For the next 15 (claims 15 - 29) the commissions is 12%. The final claim Carol processes has a 15% commission.
+The Strategic Committee is responsible to:
 
-## 4.5 Governance
+- Approve or reject Incubator rule changes (updates to this document).
+- Approve or reject quarterly budget requests from the Lead Strategist.
+- Remove any Admin other than the [Lead Strategist](#47-lead-strategist).
```

Comment:
no mechanism to remove the Lead Strategist except when he/she steps down?

Reply:
Fixed in https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/c4c94a5e12f23b7efe0e79e6b4c694488486ce9b